### PR TITLE
DOCSP-41169-1.8-release-notes

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- 1.8 Release notes

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-41169/release-notes/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-41169/api-docs/rest-api-spec/

## JIRA

https://jira.mongodb.org/browse/DOCSP-41169

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=668c40a5fb8bd13b8433480d

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,25 @@ Release Notes
...
 
+1.8.0 Changelog
+---------------
+
+*Released July 09, 2024*
+
+New features:
+
+- New conditional mapping rules to add advanced feature support for:
+  - Polymorphic relational schemas, where different rows in the same table
```
Comment (tomhollander) [inline source/release-notes.txt:21]: Markdown formatting issue; this isn't showing as a nested bullet.

File: source/release-notes.txt [hunk 2]
```diff
@@ -10,6 +10,25 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.8.0 Changelog
+---------------
+
```
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:15]: (punctuation globally)
We'll want each of these list items to end with punctuation.

File: source/release-notes.txt [hunk 3]
```diff
@@ -10,6 +10,25 @@ Release Notes
...
+*Released July 09, 2024*
+
+New features:
+
+- New conditional mapping rules to add advanced feature support for:
+  - Polymorphic relational schemas, where different rows in the same table 
+    relate to entities that need to be treated differently in the MongoDB model
+  - Relational models with foreign keys that imply “one to many” relationships 
+    but where in reality they are one-to-one and should be represented as 
+    embedded documents, not arrays
```
Comment (sarah-olson-mongodb) [inline source/release-notes.txt:25]: (simple sentences)
This is a bit of a run on sentence. Can we simplify it a little?

**Review by sarah-olson-mongodb — COMMENTED** [review]
Thanks @ianf-mongodb for getting this ready to go! Can you please take a quick editorial pass on this one before will publish.

Happy to talk it over if you want.

**Comment by davidsoulmdb** [comment]
Request: please edit the  2nd conditional mapping rule to read a variation on "Sorting and limiting of arrays, eg for subset pattern". I have updated the docsp as well for full reference

**Comment by davidsoulmdb** [comment]
Since we're fussing with the edit anyway, I'd suggest a variation on this for CMR. We've added all the names for each new CMR feature:
New conditional mapping rules for more advanced transformations:
- Mapping rule filters to include or exclude documents based on a custom Javascript expression
- Array conditions to sort and limit embedded arrays, supporting the application of the subset pattern
- Foreign key disambiguation to support tables with multiple, potentially-ambiguous references to a foreign table

**Review by tomhollander — APPROVED** [review]

**Comment by tomhollander** [comment]
Thanks @davidsoulmdb.
@ianf-mongodb please publish ASAP tomorrow; don't wait for final reviews after making David's change.

**Review by sarah-olson-mongodb — APPROVED** [review]
Nice updates @ianf-mongodb! Thanks a ton. LGTM

**Pull request merged**